cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/utils.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 the Kubeapps contributors.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	new "carvel.dev/vendir/pkg/vendir/versions/v1alpha1"
+	old "github.com/vmware-tanzu/carvel-vendir/pkg/vendir/versions/v1alpha1"
+)
+
+// This is a temporary function to convert from the new vendir version to the old
+// vendir version. This can be removed once the new vendir version is released.
+// More details at https://github.com/vmware-tanzu/kubeapps/pull/7515
```

Comment:
This PR will be partially reverted once kapp-ctrl also updates its `vendir` dep.